shared/templates/create_mount_options.py

```diff
@@ -32,7 +32,17 @@ def generate(self, target, path_info):
                         "MOUNTPOINT":       mount_point,
                         "MOUNTOPTION":       re.sub(' ', ',', option),
                     },
-                    "./ansible/mount_options{0}.yml", point_id + '_' + option_id
+                    "./ansible/mount_option{0}.yml", point_id + '_' + option_id
```

Comment:
Where they being pick up in the first place? All the OVAL/Bash files were using `mount_option` not `mount_options`.